test/db/cmd/hash

```diff
@@ -104,7 +104,7 @@ ph crc32 @ $$+10
 EOF
 EXPECT=<<EOF
 83618b8a
-f hash.crc32.83618b8a = 0x804830a
+0x804830a: crc32.83618b8a : 83618b8a
```

Comment:
uhm is /h* working like before?

Reply:
It seems `/h*` is not a correct command on master branch. What do you mean ?

Reply:
when running a search command with the * subcommand should be printing the commands to set the flags. can you add it in this pr? otherwise im fine to merge it like this and then push another pr with that up 2u

Reply:
Ok but currently `/h*` does an error. What I did in this PR is that `\h` inserts the flag directly without printing it. Before `/h` simply printed the command to add a flag.

Reply:
I know that in master /h behaves like /h* should do even if that command doesnt exist is what it should be done.

* keep your /h 
* Add /h* to print flags without setting any like /h did before

Reply:
Ok I have added it.